docs/README.md

```diff
@@ -23,6 +23,8 @@ cp ./docs/.env.example ./docs/.env
 
 Then set values to the variables.
 
+Note, if you're not using `nvm`, you'll need to use Node.js 16 and set env var with: `export NODE_OPTIONS=--openssl-legacy-provider`
```

Comment:
@bestickley Which version of Node 16 are you using? I haven't found I had to set that environment variable unless I was on Node 18